src/crispy/Note.cpp

```diff
@@ -0,0 +1,68 @@
+#include <crispy/Note.h>
+#include <crispy/assert.h>
+
+#include <gsl/gsl_assert>
```

Comment:
Can you use `<crispy/assert.h>` instead? It provides alternative implementations with a little bit more flexibility. :)